ivy/array/elementwise.py

```diff
@@ -10,11 +10,29 @@
 
 # noinspection PyUnresolvedReferences
 class ArrayWithElementwise(abc.ABC):
-    def abs(self: ivy.Array, out: Optional[ivy.Array] = None) -> ivy.Array:
+    def abs(
+        self: ivy.Array, 
+        out: Optional[ivy.Array] = None
+    ) -> ivy.Array:
         """
         ivy.Array instance method variant of ivy.abs. This method simply wraps the
         function, and so the docstring for ivy.abs also applies to this method
         with minimal changes.
+
+       Using :code:`ivy.Array` instance method:
+
+        >>> x = ivy.array([2.6, -6.6, 1.6, -0])
+        >>> y = x.abs()
+        >>> print(y)
+        ivy.array([ 2.6, 6.6, 1.6, 0])
+
+        With :code:`ivy.NativeArray` instance method:
```

Comment:
Please remove this, as we shouldn't add an instance method for `ivy.NativeArray`